src/test/resources/db/changelog/dev/qrtz01.sql

```diff
@@ -47,3 +47,22 @@ CREATE TABLE qrtz_triggers
     FOREIGN KEY (SCHED_NAME,JOB_NAME,JOB_GROUP)
         REFERENCES QRTZ_JOB_DETAILS(SCHED_NAME,JOB_NAME,JOB_GROUP)
 );
+
+
+CREATE TABLE qrtz_fired_triggers
+(
+    SCHED_NAME VARCHAR(120) NOT NULL,
+    ENTRY_ID VARCHAR(95) NOT NULL,
```

Comment:
We basically set
entryId = it.triggerName in QuickQuartzDb - acquireNextTriggers

Is this temporary? In not then, 
1. Should this then be VARCHAR(200)? same as trigger_name?
2. What about the primary key? the primary key for the qrtz_triggers table is (sched_name, trigger_name, trigger_group) But for qrtz_fired_triggers is sched_name, entry_id (which is essentially sched_name, trigger_name)

Reply:
Good call on the PK - I'll fix that up, and I think using fixed length fields is an antipattern anyway so I'll clean all those up. 

In general, these are just the "reference" schema from quartz which I'm copy/pasting for now. All of this will get cleaned up.